cognite/client/data_classes/filters.py

```diff
@@ -113,6 +115,13 @@ def load(cls, filter_: dict[str, Any]) -> Filter:
     def _filter_body(self) -> list | dict:
         ...
 
+    def _filter_names(self) -> list[str]:
```

Comment:
```suggestion
    def _involved_filter_types(self) -> set[type[Filter]]:
```
I'd suggest using the filter types here instead.